.github/workflows/test_official.yml

```diff
@@ -1,11 +1,9 @@
 name: Bot API Tests
 on:
   pull_request:
-    branches:
-      - master
-  push:
-    branches:
-      - master
```

Comment:
please re-add this in all workflows. that's what I meant with

>  I would like to see all tests running on master

Sorry if I speaking to convoluted 🙈

Reply:
But then it only runs when both push to the branch AND the files are changed. is that what you want

Reply:
mh, no.

```yml
on:
  pr:
    branches: [master]
    path: [telegram/**]
  push:
    branches: [master]
```

With this, the logic is:

```python
if (
  (pr and pr.target == "master" and pr.paths in "telegram/**")
  or (push and push.branch == master)
):
    run
```
---
```yml
on:
  pr:
    branches: [master]
  push:
    branches: [master]
```

With this, the logic is:

```python
if (
  (pr and pr.target == "master")
  or (push and push.branch == master)
):
    run
```

Note the difference in the first condition. My wish is to have the second condition (`or (push and push.branch == master)`) in all workflows :)

Reply:
but didn't I start this whole thing because we wanted to stop doing this solely based on master? Rather files.

Reply:
For PRs yes, but I disagree that tests on master should be run only a changed-files basis.